resources/js/Components/ClubPage/ContactInfo.tsx

```diff
@@ -0,0 +1,42 @@
+type ContactInfoProps = {
+    website: string;
+    email: string;
+    phone: string;
+};
+
+export default function ContactInfo({ website, email, phone }: ContactInfoProps) {
+    return (
+        <div className="contact-info border-gray-200 border bg-gray-200 max-w-xs p-6 fixed left-4 top-1/4 z-10 rounded-lg shadow-lg">
+            <h1 className="text-xl font-semibold mb-6">Contact</h1>
+
+            <div className="flex flex-col space-y-6">
+                <div className="flex items-center">
+                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5"
+                         stroke="currentColor" width="24" height="24">
+                        <path stroke-linecap="round" stroke-linejoin="round"
```

Comment:
Kunnen we alle svg dingen apart ergens stoppen. En ze importen als een component; Is absoluut niet leesbaar zo.